src/Document.php

```diff
@@ -121,7 +123,7 @@ public function index(): ?string
     public function version(): int
     {
         if (isset($this->_result)) {
-            return $this->_result->getVersion();
+            return intval($this->_result->getVersion());
```

Comment:
Elastica returns potentially a string but we always return a int. Should we adjust this as well so we support string versions as well?

Reply:
I think int casting is good enough and what we've done in the past.